app/controllers/concerns/error_handling.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module ErrorHandling
+  extend ActiveSupport::Concern
+
+  protected
+
+  def render_bad_request(resource)
+    render_bad_request_json(resource)
+  end
+
+  def render500
+    format.html { render500_html }
+    format.json { render500_json }
+  end
+
+  def render500_html
```

Comment:
Should this be here if it isn't used yet?

Reply:
That's indeed true, I didn't even think about remove the `format.html`, now done.